app/admin/pockets.rb

```diff
@@ -1,35 +1,37 @@
 ActiveAdmin.register Pocket do
-  permit_params :serial_number
+  permit_params :serial_number, :weight, :state
   actions :index, :show, :edit, :update
 
   index do
     id_column
     column :serial_number
+    column :weight
     column :state
     column :organization
     actions
   end
 
   filter :serial_number
+  filter :weight
   filter :state
   filter :organization
 
   show title: :serial_number do
     attributes_table do
-      rows :id, :serial_number, :state, :organization
+      rows :id, :serial_number, :weight, :state, :organization
     end
   end
 
   form do |f|
     f.inputs do
-      f.input :serial_number if f.object.persisted?
+      if f.object.persisted? && f.object.state != 'Unweighed'
```

Comment:
Is really necessary to check that the object is persisted? I believe that it is enough if we do not have `create` as an action.